### PR TITLE
Fix gac to work with /bin/sh that is not bash

### PR DESCRIPTION
**Pull request by fingolfin**

### CODE REVIEW

File: etc/ci.sh [hunk 1]
```diff
@@ -79,6 +79,12 @@ cd profiling
 ./configure $CONFIGFLAGS --with-gaproot=$BUILDDIR
 make V=1
 
+# Compile browse package to test gac
```
Comment (ChrisJefferson) [inline etc/ci.sh]: need a `cd ..` here

**Comment by fingolfin** [comment]
Actually, trying to test build Browse like that won't work, because it also needs a source code change. So I switched this to edim now.

**Comment by frankluebeck** [comment]
Without this PR it was not possible to `gac`-compile the kernel modules of the EDIM and Browse packages on a Debian 8 or on an Ubuntu 16.04 system (which start `/bin/sh` scripts in strict sh-compatibility mode). With this PR it works again, thanks.

I noticed that for example for the EDIM package after compilation (on Debian 8, x86_64) the `bin`
directory contains 76 kB. With GAP 4.8.6 it contains only 16 kB. It seems that in the new setup one can delete everything except the generated `*.so` file (26 kB, which in turn can be `strip`ed to 8 kB). Shouldn't the additional `.libs` directory and the `*.la` file be cleaned up by `gac`?

**Comment by fingolfin** [comment]
Yes, perhaps these should be cleared (although we might have to reconsider that when implementing `make install` -- but that's of course not an argument about cleaning these files for now, I just mention it for completeness).

However, this PR is strictly meant to fix the bug; I'd prefer to not also add other changes in here, but instead get it merged ASAP. So perhaps this could be filed as a feature request issue? Or feel free to submit a PR implementing it.

To this end, I just pushed a new revision, in an attempt to make the CI tests pass (the previous iteration failed because `gac` does not currently work on HPC-GAP, see #1295. The short term workaround is to just not test it on HPC-GAP).

**Comment by fingolfin** [comment]
The failure on Windows is another gac bug; I think I know how to fix it, will look into it when I have some time.

**Comment by fingolfin** [comment]
On Windows, we need to link the module (e.g. `edim.so` resp. `edim.dll`) against `gap.dll` (resp. `libgap.la`). The main "problem" doing that is to teach `gac` that. So we'll either need to let gac know when we are on Windows (by tweaking how we generate it), or else pass the required linker options directly to it from configure. Either works, it's a matter of taste in the end, I guess.

But I don't have time right now to work on it.

**Comment by fingolfin** [comment]
The new revision of this PR disables the edim test on Windows, too; and I created issue #1298 to log the problem.

**Comment by fingolfin** [comment]
From my POV this could be merged now.

**Pull request merged**